drivers/usb/udc/udc_dwc2.c

```diff
@@ -82,6 +76,8 @@ struct udc_dwc2_data {
 	/* Number of OUT endpoints including control endpoint */
 	uint8_t outeps;
 	uint8_t setup[8];
+	uint16_t numof_in_eps;
+	uint16_t numof_out_eps;
```

Comment:
Why `uint16_t`? The maximum value here is 16 and therefore `uint8_t` is more than enough.

Reply:
It was some leftover, removed it.